services/controller/src/plz/controller/configuration.py

```diff
@@ -86,7 +86,14 @@ def _instance_provider_from(
 
 def _images_from(config, docker_host):
     images_type = config.get('images.provider', 'local')
-    docker_api_client = docker.APIClient(base_url=docker_host)
+    docker_api_client_timeout = config.get(
+        'images.docker_api_client_timeout', None)
+    if docker_api_client_timeout is not None:
+        client_extra_args = {'timeout': docker_api_client_timeout}
```

Comment:
Why not just set it either way? `None` becomes `null`, which should work the same as not providing it at all?

Reply:
Discussed verbally